### DISCUSSION TITLE
[Kavita] Toggle for Publication Status Update via Count

### DISCUSSION THREAD

**Discussion by Sonny-Crockett** [Kareadita/Kavita#3033] [Ideas]
### Idea Description

Would it be possible to have a toggle in Kavita for ignoring updating if Publication if status is already marked "Completed"? In basically every other situation the Count logic makes sense, but typically if something is completed, its done. That could help with Kavita not over-writing komf or custom inputs.

Toggle so if people prefer or workflow supports the current Count logic, nothing changes for them, gotta be opt-in.

### Idea Category

User Experience

### Duration of Using Kavita

2 months

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by majora2007** [comment]
Since you bumped this recently, let me try to understand better. If it's marked as Complete via the API, then you have metadata that says otherwise, you want Kavita to ignore this?

**Comment by Sonny-Crockett** [comment]
Thank you for asking and I appreciate the awesome software. I am a Kavita+ user and love the external reviews and recommendations!(best features 👍)

---

So I know komf and external metadata can be kinda "fixed" by locking fields. 

So [Mangadex metadata ](https://api.mangadex.org/docs/3-enumerations/#manga-status) for example is putting that it is Complete, but from my understanding of reading the Count section in the [Kavita wiki](https://wiki.kavitareader.com/guides/metadata/comics#count), and based on that other closed issue, there can issues where the logic kicks in and assumes Completed titles are Ongoing.   

My request to this is more I guess logic based. My request is for a toggle to not trigger that Count logic if something is marked Completed from the API or User Input. As in my mind, if something has been marked Completed, that is more/less a permanent tag. I'm sure other people prefer or use a workflow that works well with the current Count logic. I have my Komf API setup fairly well so I hope I am just an outlier here. 

I mean if I just need to lock all the Publication fields, I can do that if I can figure out how to do it before it overwrites. This hiccup is the only real issue I have with the Komf + Mangadex + Kavita metadata workflow. Everything else is carrying over fine, and I can't understand (for example) how a manga from the 1960's that is very much Complete, with all chapters downloaded, gets marked as Ongoing. I know it might not matter to others but that is an important tag for me when I am choosing what to read.

**Reply by majora2007** [reply]
> As in my mind, if something has been marked Completed, that is more/less a permanent tag. 
I have a few cases where this is not true. AL considered it complete, but then the mangaka came back and started releasing again on it. Thus meaning it would need to go from Completed -> Ongoing (I handled via metadata update). 

I think that having komf lock the field for publication status once Complete might be a good way to approach this.

For your situation, feel free to create an issue with your max issue number, max volume number, library type, and count from the first issue/chapter and I can help answer your question.

**Reply by Sonny-Crockett** [reply]
Will do, I will reach out! 

And yes, for some reason ALL of my manga show as Ongoing, so separate from this, I might have something funky going on. I will try to collect the relevant information in an issue here. 

Thank you again!

**Comment by Sonny-Crockett** [comment]
https://github.com/Snd-R/komf/issues/151 - posted to Komf
